docker/hootcore/hoot-gbdx.py

```diff
@@ -50,11 +78,13 @@ def findFiles(self,inputDir,extens):
 
 
 
-    def convertFile(self,inputFile,outputFile):
+    def convertFileJson(self,envVars,inputFile,outputFile):
+        override = 'translation.override=' + json.dumps(envVars)
         hootCmd = ['hoot','convert','--error',
                 '-D','convert.ops=hoot::TranslationOp',
```

Comment:
Do you want to add the BoundingBoxVisitor here too?

Reply:
Fixed